packages/calypso-e2e/src/element-helper.ts

```diff
@@ -75,3 +75,30 @@ export async function clickNavTab( page: Page, name: string ): Promise< void > {
 	const elementHandle = await page.waitForSelector( selectors.navTab( name ) );
 	await Promise.all( [ page.waitForNavigation(), elementHandle.click() ] );
 }
+
+/**
+ * Retry any action up to three times or action passes without throwing an exception,
+ * whichever comes first.
+ *
+ * This is useful for situations where the backend must process the results of a
+ * previous action then inform the front end of the result of the process.
+ * An example of this is the user invitation system where the backend must receive,
+ * verify and send the invitation issued by a user. If the resulting checks were
+ * successful, the resulting invitation is shown as 'pending'.
+ *
+ * @param {Page} page Page object.
+ */
+export async function retryAction(
+	page: Page,
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	func: ( page: Page, ...args: any[] ) => Promise< void >
```

Comment:
I think you can take the `args` out of the signature here - since they aren't passed in otherwise, they can't be applied to function. I think people can just use closures if they want when passing it the function if they need other data!

Reply:
Fixed in 78d067d242bf8c2e0f9d2e842a9e2f4b84497b15.